Covid19Radar/Covid19Radar/Common/Utils.cs

```diff
@@ -30,6 +30,25 @@ public static DateTime[] JstDateTimes(int days)
             return dateTimes;
         }
 
+        public static string CombineAsUrl(params string[] paths)
+        {
+            if (paths is null)
```

Comment:
[修正任意]
配列が空の場合もリターンしていいと思います。

```suggestion
            if (paths is null || paths.Length == 0)
```